assets/js/script.js

```diff
@@ -1,97 +1,101 @@
 $(document).ready(function() {
-    var letter1Choice = "black"
-    var letter2Choice = "black"
+  var el = {
```

Comment:
you'll see `el` used in examples to refer to a single jquery element. I used to call this elements object `elements`, but it became a pain to type.